runtime/common/metering.go

```diff
@@ -370,28 +373,24 @@ func AdditionalAtreeMemoryUsage(originalCount uint64, elementSize uint, array bo
 	}
 }
 
-func NewArrayMemoryUsages(count uint64, elementSize uint) (MemoryUsage, MemoryUsage, MemoryUsage, MemoryUsage) {
```

Comment:
Separate the base memory usage out, so it is always metered, not just when atree memory metering is necessary